### ISSUE TITLE
[Feedback]Installing AdGuard Not Opening

### ISSUE THREAD

**Issue by egrogg** [IceWhaleTech/CasaOS#1119]
### Description

hello, I have installed the AdGuard Home from the AppStore. I was able to set it up and then it does not open to the dashboard once setup says it is complete. By the way i just got the ZimaBoard today.

screenshots.

![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/9f8e2528-6cfd-464a-ac43-cca19bed051a)
![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/ce936b81-c901-4376-bfeb-9811dc96550b)

### Additional Information

	 - OS: linux
	 - CasaOS Version: 0.4.3
	 - Disk Total: 27712 
	 - Disk Used: 8410 
	 - System Info: [Debian GNU/Linux 11   \l Bit:64 Mem:7789M Core:4 Linux CasaOS 5.10.0-22-amd64 #1 SMP Debian 5.10.178-3 (2023-04-22) x86_64 GNU/Linux]
	 - Remote Version: 0.4.3
	 - Browser: Firefox 
	 - Version: 113.0

**Comment by egrogg** [comment]
sorry, here are the logs;

2023/05/23 21:28:33.832998 [info] AdGuard Home, version v0.107.5
2023/05/23 21:28:33.833215 [info] This is the first time AdGuard Home is launched
2023/05/23 21:28:33.833242 [info] Checking if AdGuard Home has necessary permissions
2023/05/23 21:28:33.833653 [info] AdGuard Home can bind to port 53
2023/05/23 21:28:33.835740 [info] Initializing auth module: /opt/adguardhome/work/data/sessions.db
2023/05/23 21:28:33.838388 [info] auth: initialized.  users:0  sessions:0
2023/05/23 21:28:33.838474 [info] Initialize web module
2023/05/23 21:28:33.838507 [info] This is the first launch of AdGuard Home, redirecting everything to /install.html 
2023/05/23 21:28:33.838563 [info] AdGuard Home is available at the following addresses:
2023/05/23 21:28:33.838976 [info] Go to http://127.0.0.1:3000
2023/05/23 21:28:33.838997 [info] Go to http://172.17.0.2:3000
2023/05/23 21:29:15.765498 [info] Starting the DNS proxy server
2023/05/23 21:29:15.765558 [info] Ratelimit is enabled and set to 20 rps
2023/05/23 21:29:15.765566 [info] The server is configured to refuse ANY requests
2023/05/23 21:29:15.765574 [info] DNS cache is enabled
2023/05/23 21:29:15.765588 [info] MaxGoroutines is set to 300
2023/05/23 21:29:15.765612 [info] Creating the UDP server socket
2023/05/23 21:29:15.765814 [info] Listening to udp://[::]:53
2023/05/23 21:29:15.765830 [info] Creating a TCP server socket
2023/05/23 21:29:15.765893 [info] Listening to tcp://[::]:53
2023/05/23 21:29:15.766029 [info] Entering the UDP listener loop on [::]:53
2023/05/23 21:29:15.766113 [info] Entering the tcp listener loop on [::]:53
2023/05/23 21:29:15.951830 [info] Go to http://127.0.0.1:80
2023/05/23 21:29:15.951865 [info] Go to http://172.17.0.2:80
2023/05/23 21:29:16.183317 [info] saving filter 1 contents to: /opt/adguardhome/work/data/filters/1.txt
2023/05/23 21:29:16.183530 [info] updated filter 1: 1055833 bytes, 52673 rules
2023/05/23 21:29:16.183559 [info] Updated filter #1.  Rules: 0 -> 52673

**Comment by WisdomSky** [comment]
can you show us a screenshot of the adguard settings?

like this:
![image](https://github.com/IceWhaleTech/CasaOS/assets/6792172/63c60723-7196-46d8-b3a2-913dc1480453)

**Comment by raller1028** [comment]
Can you try changing the webui port to 3000, my AdGuard settings are as follows and I can open it normally
![image](https://github.com/IceWhaleTech/CasaOS/assets/57336867/af97fb22-8a5a-4dbe-b448-15ea57d918e0)
![image](https://github.com/IceWhaleTech/CasaOS/assets/57336867/e6c81d6e-e5c2-46d8-9e34-bf4ab51b2a86)
You can also try importing this AdGuardHome.json that I provided in the custom installation

**Comment by raller1028** [comment]
[AdGuardHome.zip](https://github.com/IceWhaleTech/CasaOS/files/11551046/AdGuardHome.zip)

**Comment by egrogg** [comment]
Hello, 
Sorry I have not responded sooner.


I removed the current install and installed the json that was forwarded to me.  Still not able to get to the install?

I don't know if it has anything to do with it, but I SSH'd into the box after I got it and did an update and upgrade then I updated the CasaOS (from the gui) to the new version...then installing AdGuard doesn't work.  Maybe an update issue? Also as a note I did try the pi-hole.  Pi-Hole installs but nothing will connect to it?  I have a pi-hole currently on rasberry pi but some end points are not on it, so i tried adding them to this pi-hole and nothing?  No one connects. I am sure I can ping the box, and can connect to the CasaOS gui so, not sure where the issue would be? Docker?  


Here are the current settings for the AdGuard:

![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/637ba5f8-35b8-42d3-b0fe-571a0a4493a5)
![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/331c5f84-99d5-481f-8165-271e4c64a47e)
![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/1ac72a35-96e8-4669-ad0d-8cc1f3c4fd4b)

Logs:
2023/05/24 19:31:53.304569 [info] AdGuard Home, version v0.107.5
2023/05/24 19:31:53.304669 [info] This is the first time AdGuard Home is launched
2023/05/24 19:31:53.304703 [info] Checking if AdGuard Home has necessary permissions
2023/05/24 19:31:53.305058 [info] AdGuard Home can bind to port 53
2023/05/24 19:31:53.307989 [info] Initializing auth module: /opt/adguardhome/work/data/sessions.db
2023/05/24 19:31:53.337959 [info] auth: initialized.  users:0  sessions:0
2023/05/24 19:31:53.338030 [info] Initialize web module
2023/05/24 19:31:53.338061 [info] This is the first launch of AdGuard Home, redirecting everything to /install.html 
2023/05/24 19:31:53.338121 [info] AdGuard Home is available at the following addresses:
2023/05/24 19:31:53.338617 [info] Go to http://127.0.0.1:3000
2023/05/24 19:31:53.338640 [info] Go to http://172.17.0.2:3000
2023/05/24 19:32:15.677908 [info] Starting the DNS proxy server
2023/05/24 19:32:15.677978 [info] Ratelimit is enabled and set to 20 rps
2023/05/24 19:32:15.677987 [info] The server is configured to refuse ANY requests
2023/05/24 19:32:15.677997 [info] DNS cache is enabled
2023/05/24 19:32:15.678013 [info] MaxGoroutines is set to 300
2023/05/24 19:32:15.678039 [info] Creating the UDP server socket
2023/05/24 19:32:15.678220 [info] Listening to udp://[::]:53
2023/05/24 19:32:15.678238 [info] Creating a TCP server socket
2023/05/24 19:32:15.678310 [info] Listening to tcp://[::]:53
2023/05/24 19:32:15.678599 [info] Entering the UDP listener loop on [::]:53
2023/05/24 19:32:15.679042 [info] Entering the tcp listener loop on [::]:53
2023/05/24 19:32:15.855816 [info] Go to http://127.0.0.1:80
2023/05/24 19:32:15.856202 [info] Go to http://172.17.0.2:80
2023/05/24 19:32:16.109034 [info] saving filter 1 contents to: /opt/adguardhome/work/data/filters/1.txt
2023/05/24 19:32:16.109649 [info] updated filter 1: 1058492 bytes, 52807 rules
2023/05/24 19:32:16.109696 [info] Updated filter #1.  Rules: 0 -> 52807


Here is the docker images from SSH

casaos@CasaOS:~$ sudo docker ps -a
[sudo] password for casaos:
CONTAINER ID   IMAGE                          COMMAND                  CREATED          STATUS          PORTS                                                                                                                                                                                                                                                                                                          NAMES
0ceec8782339   adguard/adguardhome:v0.107.5   "/opt/adguardhome/Ad…"   12 minutes ago   Up 12 minutes   80/tcp, 67-68/udp, 0.0.0.0:53->53/udp, :::53->53/udp, 443/udp, 0.0.0.0:53->53/tcp, :::53->53/tcp, 853/udp, 0.0.0.0:853->853/tcp, :::853->853/tcp, 443/tcp, 3001/tcp, 3000-3001/udp, 5443/tcp, 5443/udp, 0.0.0.0:3000->3000/tcp, 0.0.0.0:784->784/udp, :::3000->3000/tcp, :::784->784/udp, 8853/udp, 6060/tcp   AdGuardHome
casaos@CasaOS:~$


casaos@CasaOS:~$ sudo docker images -a
REPOSITORY            TAG        IMAGE ID       CREATED         SIZE
adguard/adguardhome   v0.107.5   452a671b0bf9   14 months ago   80.7MB
casaos@CasaOS:~$

I don't understand why that says 14 Months ago?  that is a strange one?



and here is the status of docker:

asaos@CasaOS:~$ sudo systemctl status docker
● docker.service - Docker Application Container Engine
     Loaded: loaded (/lib/systemd/system/docker.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-05-23 17:27:42 EDT; 22h ago
TriggeredBy: ● docker.socket
       Docs: https://docs.docker.com
   Main PID: 1360 (dockerd)
      Tasks: 112
     Memory: 274.9M
        CPU: 1min 37.714s
     CGroup: /system.slice/docker.service
             ├─  1360 /usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
             ├─152171 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 3000 -container-ip 172.17.0.2 -container-port 3000
             ├─152180 /usr/bin/docker-proxy -proto tcp -host-ip :: -host-port 3000 -container-ip 172.17.0.2 -container-port 3000
             ├─152195 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 853 -container-ip 172.17.0.2 -container-port 853
             ├─152201 /usr/bin/docker-proxy -proto tcp -host-ip :: -host-port 853 -container-ip 172.17.0.2 -container-port 853
             ├─152217 /usr/bin/docker-proxy -proto udp -host-ip 0.0.0.0 -host-port 784 -container-ip 172.17.0.2 -container-port 784
             ├─152225 /usr/bin/docker-proxy -proto udp -host-ip :: -host-port 784 -container-ip 172.17.0.2 -container-port 784
             ├─152239 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 53 -container-ip 172.17.0.2 -container-port 53
             ├─152246 /usr/bin/docker-proxy -proto tcp -host-ip :: -host-port 53 -container-ip 172.17.0.2 -container-port 53
             ├─152261 /usr/bin/docker-proxy -proto udp -host-ip 0.0.0.0 -host-port 53 -container-ip 172.17.0.2 -container-port 53
             └─152266 /usr/bin/docker-proxy -proto udp -host-ip :: -host-port 53 -container-ip 172.17.0.2 -container-port 53

May 23 17:27:42 CasaOS dockerd[1360]: time="2023-05-23T17:27:42.286768525-04:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can>
May 23 17:27:42 CasaOS dockerd[1360]: time="2023-05-23T17:27:42.416037446-04:00" level=info msg="Loading containers: done."
May 23 17:27:42 CasaOS dockerd[1360]: time="2023-05-23T17:27:42.559596514-04:00" level=info msg="Docker daemon" commit=463850e graphdriver=overlay2 version=24.0.1
May 23 17:27:42 CasaOS dockerd[1360]: time="2023-05-23T17:27:42.560529890-04:00" level=info msg="Daemon has completed initialization"
May 23 17:27:42 CasaOS dockerd[1360]: time="2023-05-23T17:27:42.633227821-04:00" level=info msg="API listen on /run/docker.sock"
May 23 17:27:42 CasaOS systemd[1]: Started Docker Application Container Engine.
May 23 17:50:23 CasaOS dockerd[1360]: time="2023-05-23T17:50:23.041852919-04:00" level=info msg="ignoring event" container=38d88284e15be44ed0ec00b824f1c23f8c6bc13685d743928df0b3b25e9984a6 mod>
May 23 17:50:23 CasaOS dockerd[1360]: time="2023-05-23T17:50:23.063469446-04:00" level=warning msg="ShouldRestart failed, container will not be restarted" container=38d88284e15be44ed0ec00b824>
May 23 18:10:43 CasaOS dockerd[1360]: time="2023-05-23T18:10:43.137234625-04:00" level=info msg="ignoring event" container=c3c0176ddf6141e2cdd435c4d8dd4dadd5bdc309b03a279584c43d5d2b6f5bcc mod>
May 23 18:10:43 CasaOS dockerd[1360]: time="2023-05-23T18:10:43.158609260-04:00" level=warning msg="ShouldRestart failed, container will not be restarted" container=c3c0176ddf6141e2cdd435c4d8>
lines 1-32/32 (END)

**Comment by egrogg** [comment]
the attached is the Yaml file from the conf directory. Don't know if all this I am posting will help or not but I hope so.

![image](https://github.com/IceWhaleTech/CasaOS/assets/118689943/99fa3967-828a-491e-9882-14f89327a8c2)

[AdGuardHomeYaml.zip](https://github.com/IceWhaleTech/CasaOS/files/11559095/AdGuardHomeYaml.zip)

**Comment by raller1028** [comment]
@egrogg Sorry, I did some more research on how to use AdGuardHome correctly and the problem is that there is no port 80 mapping in this json. Here's the solution for you: After you've finished setting it up, add the port 80 mapping and change the webui port to 8080 (as port 80 is already occupied by casaos)
With a video of my test
[2023-05-25 11-20-41.zip](https://github.com/IceWhaleTech/CasaOS/files/11560924/2023-05-25.11-20-41.zip)

**Comment by silversword411** [comment]
TLDR. Add this. 

80 should be setup by default, not sure why it's not and using 3000 instead

![2023-11-10_092428 - adguard](https://github.com/IceWhaleTech/CasaOS/assets/13395348/c53682f1-2937-40bc-b545-138a019d5707)

**Comment by YordanMitev** [comment]
> TLDR. Add this.
> 
> 80 should be setup by default, not sure why it's not and using 3000 instead
> 
> ![2023-11-10_092428 - adguard](https://private-user-images.githubusercontent.com/13395348/282088343-c53682f1-2937-40bc-b545-138a019d5707.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzgyNzksIm5iZiI6MTcwNjg3Nzk3OSwicGF0aCI6Ii8xMzM5NTM0OC8yODIwODgzNDMtYzUzNjgyZjEtMjkzNy00MGJjLWI1NDUtMTM4YTAxOWQ1NzA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAyVDEyNDYxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5ZDlhZTk3OTA3YWU4MjUyMzQxODJlNmU4NDYzOGZiZGQ4ODBmNmQyNDNmMzg5Njc0Mjc3NTdjYWE3MGJmZjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.a4EOsl7s_G7qk9zB2IewLuNjnjkE2T9p_dlf6c19mbI)

This is working solution about me!!! Thx alot

**Comment by alptugan** [comment]
> TLDR. Add this.
> 
> 80 should be setup by default, not sure why it's not and using 3000 instead
> 
> ![2023-11-10_092428 - adguard](https://private-user-images.githubusercontent.com/13395348/282088343-c53682f1-2937-40bc-b545-138a019d5707.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY1MDU0NzUsIm5iZiI6MTcxNjUwNTE3NSwicGF0aCI6Ii8xMzM5NTM0OC8yODIwODgzNDMtYzUzNjgyZjEtMjkzNy00MGJjLWI1NDUtMTM4YTAxOWQ1NzA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIzVDIyNTkzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5Y2IzNzNhODg2NGRmNjRkYWRlNTBmN2UzZmU0M2Y1MzBhZDFjMTg5NWVjZDhlYTllNTRmYWQ0NjljMDc2ZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vCZqc_uzXeK6Ly6ViY89lXNU7U7fI57Sn7gUBCDbWK4)

Still works
